### PR TITLE
Setup MTU for iOS

### PR DESCRIPTION
**Pull request by isamnezia**

### CODE REVIEW

File: client/vpnconnection.cpp [hunk 1]
```diff
@@ -305,6 +305,11 @@ QJsonObject VpnConnection::createVpnConfiguration(int serverIndex, const ServerC
     // TODO: try to get hostName, port, description for 3rd party configs
     // vpnConfiguration[config_key::port] = ...;
 
+#if defined Q_OS_IOS
+    QJsonObject awg = server[config_key::containers].toArray().first().toObject()[config_key::awg].toObject();
```
Comment (outspace) [inline client/vpnconnection.cpp]: What about WG config? Does it apply to WG config?

Comment (isamnezia) [reply client/vpnconnection.cpp]: There was no mtu field in wg json. And now there is no mtu field in openvpn json.

Comment (outspace) [reply client/vpnconnection.cpp]: For WG config it has same logic to AWG config. Please try to change mtu for WG protocol and check that field.

Comment (isamnezia) [reply client/vpnconnection.cpp]: I know about same logic. I mean wg/awg. I will write to you in tg.

**Pull request merged**